scripts/wsl2_ai_tuning.py

```
@@ -0,0 +1,401 @@
+        res = subprocess.run(
+            ["sysctl", "-n", param],
+            capture_output=True,
+            text=True,
+            check=True,
+        )
+        return int(res.stdout.strip())
+    except Exception:
+        return 0
+
+
+def apply_tuning(wsl_mem_gb, wsl_cpus, networking_mode, distro_family):
```

Ruff (0.16.1)

[warning] Too many branches (15 > 12)

(PLR0912)